components/widgets/CurrentData/patterns/Precipitation/index.js

```diff
@@ -2,34 +2,34 @@ import PropTypes from "prop-types";
 import { useTranslation } from "react-i18next";
 import { formatPercent } from "helpers/formatters";
 import WidgetBackground from "@/components/atomic/WidgetBackground";
-import * as Styled from "./styles";
+import * as Styled from "../../styles";
 
 const PrecipitationCurrent = ({ precipitation = 0, humidity = 0 }) => {
   const { t, i18n } = useTranslation();
   const { language = "en" } = i18n;
 
   return (
     <WidgetBackground $variant="secondary">
-      <Styled.WidgetValue
+      <Styled.Value
```

Comment:
unify all the primitives for current data widgets into a single shared set, and implement widgets as patterns